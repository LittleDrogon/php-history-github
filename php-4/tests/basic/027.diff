001- array(4) {
001+ array(5) {
006-   ["d"]=>
006+   ["c"]=>
007-   string(3) "123"
007+   array(1) {
008-   ["e"]=>
008+     [0]=>
009-   array(1) {
009+     array(1) {
010-     [0]=>
010+       [0]=>
011-     array(1) {
011+       array(1) {
012-       [0]=>
012+         [0]=>
013-       string(1) "3"
013+         array(1) {
014-     }
014+           [0]=>
015-   }
015+           array(1) {
016- }
016+             [0]=>
017- string(124) "Unknown: Input variable nesting level exceeded 10. To increase the level change max_input_nesting_level in php.ini."
017+             array(1) {
018+               [0]=>
019+               array(1) {
020+                 [0]=>
021+                 array(1) {
022+                   [0]=>
023+                   array(1) {
024+                     [0]=>
025+                     array(1) {
026+                       [0]=>
027+                       array(1) {
028+                         [0]=>
029+                         array(1) {
030+                           [0]=>
031+                           array(1) {
032+                             [0]=>
033+                             array(1) {
034+                               [0]=>
035+                               array(1) {
036+                                 [0]=>
037+                                 array(1) {
038+                                   [0]=>
039+                                   array(1) {
040+                                     [0]=>
041+                                     array(1) {
042+                                       [0]=>
043+                                       array(1) {
044+                                         [0]=>
045+                                         array(1) {
046+                                           [0]=>
047+                                           array(1) {
048+                                             [0]=>
049+                                             array(1) {
050+                                               [0]=>
051+                                               string(3) "123"
052+                                             }
053+                                           }
054+                                         }
055+                                       }
056+                                     }
057+                                   }
058+                                 }
059+                               }
060+                             }
061+                           }
062+                         }
063+                       }
064+                     }
065+                   }
066+                 }
067+               }
068+             }
069+           }
070+         }
071+       }
072+     }
073+   }
074+   ["d"]=>
075+   string(3) "123"
076+   ["e"]=>
077+   array(1) {
078+     [0]=>
079+     array(1) {
080+       [0]=>
081+       string(1) "3"
082+     }
083+   }
084+ }
085+ NULL