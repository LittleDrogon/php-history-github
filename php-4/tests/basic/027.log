
---- EXPECTED OUTPUT
array(4) {
  ["a"]=>
  string(1) "1"
  ["b"]=>
  string(3) "ZYX"
  ["d"]=>
  string(3) "123"
  ["e"]=>
  array(1) {
    [0]=>
    array(1) {
      [0]=>
      string(1) "3"
    }
  }
}
string(124) "Unknown: Input variable nesting level exceeded 10. To increase the level change max_input_nesting_level in php.ini."
---- ACTUAL OUTPUT
array(5) {
  ["a"]=>
  string(1) "1"
  ["b"]=>
  string(3) "ZYX"
  ["c"]=>
  array(1) {
    [0]=>
    array(1) {
      [0]=>
      array(1) {
        [0]=>
        array(1) {
          [0]=>
          array(1) {
            [0]=>
            array(1) {
              [0]=>
              array(1) {
                [0]=>
                array(1) {
                  [0]=>
                  array(1) {
                    [0]=>
                    array(1) {
                      [0]=>
                      array(1) {
                        [0]=>
                        array(1) {
                          [0]=>
                          array(1) {
                            [0]=>
                            array(1) {
                              [0]=>
                              array(1) {
                                [0]=>
                                array(1) {
                                  [0]=>
                                  array(1) {
                                    [0]=>
                                    array(1) {
                                      [0]=>
                                      array(1) {
                                        [0]=>
                                        array(1) {
                                          [0]=>
                                          array(1) {
                                            [0]=>
                                            array(1) {
                                              [0]=>
                                              string(3) "123"
                                            }
                                          }
                                        }
                                      }
                                    }
                                  }
                                }
                              }
                            }
                          }
                        }
                      }
                    }
                  }
                }
              }
            }
          }
        }
      }
    }
  }
  ["d"]=>
  string(3) "123"
  ["e"]=>
  array(1) {
    [0]=>
    array(1) {
      [0]=>
      string(1) "3"
    }
  }
}
NULL
---- FAILED
