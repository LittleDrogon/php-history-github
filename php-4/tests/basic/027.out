array(5) {
  ["a"]=>
  string(1) "1"
  ["b"]=>
  string(3) "ZYX"
  ["c"]=>
  array(1) {
    [0]=>
    array(1) {
      [0]=>
      array(1) {
        [0]=>
        array(1) {
          [0]=>
          array(1) {
            [0]=>
            array(1) {
              [0]=>
              array(1) {
                [0]=>
                array(1) {
                  [0]=>
                  array(1) {
                    [0]=>
                    array(1) {
                      [0]=>
                      array(1) {
                        [0]=>
                        array(1) {
                          [0]=>
                          array(1) {
                            [0]=>
                            array(1) {
                              [0]=>
                              array(1) {
                                [0]=>
                                array(1) {
                                  [0]=>
                                  array(1) {
                                    [0]=>
                                    array(1) {
                                      [0]=>
                                      array(1) {
                                        [0]=>
                                        array(1) {
                                          [0]=>
                                          array(1) {
                                            [0]=>
                                            array(1) {
                                              [0]=>
                                              string(3) "123"
                                            }
                                          }
                                        }
                                      }
                                    }
                                  }
                                }
                              }
                            }
                          }
                        }
                      }
                    }
                  }
                }
              }
            }
          }
        }
      }
    }
  }
  ["d"]=>
  string(3) "123"
  ["e"]=>
  array(1) {
    [0]=>
    array(1) {
      [0]=>
      string(1) "3"
    }
  }
}
NULL